src/index.css

```
@@ -97,3 +97,30 @@
     @apply bg-background text-foreground;
   }
 }
+
+body.compact-mode {
+  --spacing-unit: 0.75;
+  font-size: 0.875rem;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)